src/components/Main.js

```diff
@@ -124,6 +127,18 @@ const styles = StyleSheet.create({
     shadowOpacity: 0.5,
     shadowRadius: 3,
   },
+  comingView: {
```

Comment:
How about change name to 'comingSoon...'? I couldn't understand the meaning of it at first